_includes/scenarios/integrate/updating-identityattributes-of-yourself.md

```diff
@@ -1,17 +1,45 @@
 
-Each peer would receive a Notification containing an AttributeSucceededNotificationItem which is automatically processed by each Identity. Thus, succeeding Attribute does not result into decidable Requests for the peer. They are only informed about the change.
+Next, you can [notify your peers about the succession]({% link _docs_use-cases/use-case-consumption-notify-peer-about-identityattribute-succession.md %}), so that they'll receive the current version, too.
+Note that it is also possible to make a selection of the peers you want to notify about the Attribute succession, if for some reason you don't want to inform all of them.
+The own shared IdentityAttributes associated with the peers chosen will be succeeded in the same manner than your unshared IdentityAttribute before.
+Moreover, the peers will receive a Notification, such that their instances replicate the succession chain at your side.
 
-# Examples
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/aef8ef85-ca90-47ae-959b-5bab7af7f43a" id="8.LThfly522r"></iframe></div>
 
-- Changing IdentityAttributes because of name changes, a move to a new address or change of communication details
-- An Identity receiving a new telephone number but does not want to share this new telephone number to every peer
+## Succeeding own RelationshipAttributes
+
+RelationshipAttributes are, as the name suggests, always associated with a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship).
+Thus, it is not possible to have unshared instances of them.
+Instead, wanting to [create a RelationshipAttribute]({% link _docs_use-cases/use-case-consumption-create-and-share-a-relationshipattribute.md %}) always involves sharing it directly with the peer.
+So, a Request containing a [CreateAttributeRequestItem]({% link _docs_integrate/requests-and-requestitems.md %}#createattributerequestitem) with the content of the RelationshipAttribute will be created and will be sent via [Message]({% link _docs_integrate/data-model-overview.md %}#message) to the peer.
+Only if the peer accepts this Request, an instance of the RelationshipAttribute will be created at their side and, by receiving the [Response]({% link _docs_integrate/data-model-overview.md %}#response), at your side, too.
+Hence, you and your peer will always have an identical instance of the RelationshipAttribute, only differing in the `peer` property.
+
+Equally to IdentityAttributes, again only the owner can [succeed a RelationshipAttribute]({% link _docs_use-cases/use-case-consumption-succeed-a-relationshipattribute-and-notify-peer.md %}).
+Doing so, the `succeededBy` property of the predecessor and the `succeeds` property of the successor are set.
+Then, a Notification is sent to the peer, which handles the replication of the succession chain at their side.
```

Comment:
Could state that in the case of RelationshipAttribute a notification is sent immediately upon succeeding an attribute, i.e. this whole keep-my-peers-up-to-date process is automatic.